src/Alerts/atom/Modal.tsx

```diff
@@ -16,7 +16,10 @@ const Container = styled.div`
   align-items: center;
   justify-content: center;
   background-color: hsl(0, 0%, 0%, 0.2);
+  -webkit-backdrop-filter: blur(5px);
+  backdrop-filter: blur(5px);
```

Comment:
This is way easier than the method we had to use before! I'm glad this has better browser support at last! 🥳